Workbooks/Azure Active Directory TroubleShooting/SignInFailureAnalysis/SignInFailureAnalysis.workbook

```diff
@@ -229,190 +238,121 @@
-        "aggregation": 0,
-        "showAnnotations": false,
-        "showAnalytics": false,
+        "query": "SigninLogs \r\n|where AppDisplayName in ({Apps}) or '*' in ({Apps})\r\n| extend ErrorCode = tostring(Status.errorCode) \r\n| extend FailureReason = Status.failureReason \r\n| where ErrorCode in (\"50058\",\"50140\", \"51006\", \"50059\", \"65001\", \"52004\", \"50055\", \"50144\",\"50072\", \"50074\", \"16000\",\"16001\", \"16003\", \"50127\", \"50125\", \"50129\",\"50143\", \"81010\", \"81014\", \"81012\") \r\n| where '{InterruptErrorCode}' == '*' or '{InterruptErrorCode}' == ErrorCode\r\n| top 200 by TimeGenerated desc\r\n| extend TimeFromNow = now() - TimeGenerated\r\n| extend TimeAgo = strcat(case(TimeFromNow < 2m, strcat(toint(TimeFromNow / 1m), ' seconds'), TimeFromNow < 2h, strcat(toint(TimeFromNow / 1m), ' minutes'), TimeFromNow < 2d, strcat(toint(TimeFromNow / 1h), ' hours'), strcat(toint(TimeFromNow / 1d), ' days')), ' ago')\r\n| project Identity, ['❌ Error Code'] = ErrorCode, ['Sign-in Time'] = TimeAgo\r\n",
+        "size": 1,
+        "showAnalytics": true,
+        "title": "Sign-ins waiting on user action",
         "timeContext": {
-          "durationMs": 0,
-          "endTime": null,
-          "createdTime": "2019-02-07T17:54:26.076Z",
-          "isInitialTime": false,
-          "grain": 1,
-          "useDashboardTimeRange": false
+          "durationMs": 0
         },
         "timeContextFromParameter": "TimeRange",
+        "queryType": 0,
         "resourceType": "microsoft.operationalinsights/workspaces",
-        "visualization": "table",
         "gridSettings": {
           "formatters": [
             {
-              "columnMatch": "Id",
-              "formatter": 5,
-              "formatOptions": {}
-            },
-            {
-              "columnMatch": "Sign-in Count",
+              "columnMatch": "Interrupt Count",
               "formatter": 8,
               "formatOptions": {
                 "min": 0,
-                "palette": "coldHot"
-              }
-            },
-            {
-              "columnMatch": "Trend",
-              "formatter": 9,
-              "formatOptions": {
-                "min": 0,
-                "palette": "coldHot"
+                "palette": "orange"
               }
-            },
-            {
-              "columnMatch": "ParentId",
-              "formatter": 5,
-              "formatOptions": {}
             }
           ],
-          "hierarchySettings": {
-            "idColumn": "ParentId",
-            "parentColumn": "Id",
-            "treeType": 0,
-            "expanderColumn": "Name"
-          }
+          "filter": true
         }
       },
-      "conditionalVisibility": null
+      "conditionalVisibility": null,
+      "customWidth": "33",
+      "showPin": true,
+      "name": "query - 7 - Copy"
     }
-  ]
+  ],
+  "$schema": "https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/schema/workbook.json"
```

Comment:
I see @schema is at the bottom but to be consist with others, would it be better to put it at the top?

Reply:
This is the default in the editor now. Users shouldn't have to edit the json manually. Schema is not a big deal - I don't fret about consistency here. Even if it was a big deal, it should be fixed in the serialization (users shouldn't have to hand edit this complex json).